docs/content/guides/author-apps/kubernetes/how-to-kubernetes-resource/index.md

```diff
@@ -0,0 +1,118 @@
+---
+type: docs
+title: "How-To: Use a Kubernetes resources in your application"
+linkTitle: "Use a Kubernetes resources in your application"
+description: "Learn how to use a Kubernetes resources in your application"
+weight: 200
+categories: "How-To"
+tags: ["Kubernetes"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Leverage Kubernetes resources in your Radius Application directly.
+
+## Prerequisites
+
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Step 1: Start with a container
+
+Begin by creating a file named `app.bicep` with a Radius [container]({{< ref "guides/author-apps/containers" >}}):
+
+{{< rad file="snippets/app.bicep" embed=true >}}
+
+## Step 2: Define the Kubernetes provider
+
+Add the following import statement and define your Kubernetes [`namespace` property]({{< ref "/guides/operations/kubernetes/overview#namespace-mapping" >}}) as well as the [`kubeConfig` property](https://kubernetes.io/docs/reference/config-api/kubeconfig.v1/). The 
```

Comment:
```suggestion
At the top of your Bicep file, import the `kubernetes` provider and add its configuration. The `namespace` property determines where to deploy Kubernetes resources by default. The `kubeConfig` property is not currently used and can remain as an empty string (`''`):
```

Updated the step to better describe the provider and the properties. Note the namespace page you linked to is a separate concept.